### PR TITLE
Add prettier format check to CI

### LINKED ISSUE #967: Add prettier format checking to CI

### PR DESCRIPTION
**Pull request by GarvinH**
closes #967

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -272,7 +287,6 @@ jobs:
       # repeats some of the server tests, because these fail fast and short-curcuit us before the long-running browser tests
       - run: (cd server && npm run unit_tests)
       
-      - run: npm run check_format client
```
Comment (GarvinH) [inline .circleci/config.yml:275]: Note/Reasoning: for whatever reason, this doesn't doesn't actually throw an error

**Comment by ktw1016** [comment]
Will you be fixing the files that fail so the test passes?

**Comment by GarvinH** [comment]
Yea, I wasn't sure if we wanted it on a separate PR or not. But I'll do it now

**Review by ktw1016 — APPROVED** [review]
👍

**Pull request merged**